roles/lib_utils/src/class/yedit.py

```diff
@@ -1,5 +1,5 @@
 # flake8: noqa
-# pylint: skip-file
+# pylint: disable=undefined-variable,missing-docstring
```

Comment:
I think there will be other issues with pylint if we don't skip the src files for now.